pitch.md

```diff
@@ -1,60 +1,224 @@
+| …     | …      | …                    | …                        |
+| 247.0 | 86     | Blackout             | Audrey: "…all was well." |      
+
+For a standard production there will likely exist such a cue sheet independently
+for scenic transitions, lights, and sound. The stage manager merges all the cues
+into the prompt book's script and works with the designers to move their exact
+position around as the designers watch the production rehearse.
+
+Not only is this inconvenient, but it can create a disconnect between what the
+designer thinks the cue is and what finally ends up in the prompt book; as they
+adjust cues, if they are not careful to update their respective cue sheets, the
+individual cues may be out of sync.
+
+By creating a digital and semantic way to describe the cues, individual "cue sheets"
+could be automatically merged into a digital script and allow for edits to be
+propogated to/from the prompt book layout and the individual cue sheets. With
+a fully developed system, cues could be edited in realtime by multiple people
+and be searchable. There would be no need to flip through the script to see where
+that cue at line "…all was well" occurs or what it says. During a rehearsal, if
+a designer says take it back to cue 47, the stage manager would instantly be able
+to find the page saving time and page flipping. In an ideal world, a nice GUI
+would allow the stage manager and production team to easily interact with the
+script and generate individualized cue sheets.
 
 ## Why a language?
 _Why is a **DSL** appropriate for your user(s)? How does it address the need?_
 
+Cue writing is inherently technical and should be written in a procedural way;
+the procedural aspect supports my idea that a DSL is appropriate for the task
+of cue writing. With a simple, expressive, and concise language, I think many
+designers would agree that this could be an appropriate way to author and edit
+cues.
+
+Instead of using multiple Excel sheets in varying systematic formats, we can
+express the same cues concisely and reliably in a DSL that can allow designers
+and stage managers to interact with the cues more efficiently and flexibly.
+
+I will admit, a language that you type out is not the optimal way to compose
+the cues; a GUI paired with the DSL, would bring this project to a level who's
+convenience and ease of use would make it practical solution for stage managers
+and designers.
 
 ## Why you?
 _What excites you about this idea? How did you come up with it?_
 
+As a stage manager who has worked on several large productions involving hundreds
+of cues, I've always dreaded transferring designer cue sheets' cues into the
+prompt book. I've also longed for a way to be able to search and jump to cues
+without spending time searching through the script.
+
+For the past several shows I've worked on, I've debated making a digital prompt
+book, but was dissapointed with the current solutions&mdash;annotating a PDF.
```

Comment:
PDF annotators are cumbersome and require manually inputting the cue in the correct location. The DSL would provide a systematic way of describing this placement among other structured data like which character's or people are involved, timing, etc.